react/features/base/tracks/functions.js

```diff
@@ -86,6 +91,7 @@ export function createLocalTracksF(
                     effects,
                     firefox_fake_device, // eslint-disable-line camelcase
                     micDeviceId,
+                    presenterCameraConstraints,
```

Comment:
I assume this requires a change in lib-jitsi-meet? Where is the PR for it? Ideally we shouldn't need this and the track is just a regular video track to which we pass some different constraints, and then attach some extra indicator to mark it as the "presenter" track.

Reply:
Modified createLocalTracksF to use the constraints that are passed.